crates/circuit/src/dag_circuit.rs

```diff
@@ -4383,8 +4439,8 @@ impl DAGCircuit {
     /// Add a captured stretch to the circuit.
     ///
     /// Args:
-    ///     var: the stretch to add.
-    fn add_captured_stretch(&mut self, py: Python, var: &Bound<PyAny>) -> PyResult<()> {
+    ///     stretch: the stretch to add.
+    fn add_captured_stretch(&mut self, py: Python, stretch: &Bound<PyAny>) -> PyResult<()> {
```

Comment:
This is actually a breaking api change right? Python's arguments by default are both positional and by name, and we didn't declare the signature of this pymethod to just deal with this positionally the name can be used to do `dag.add_captured_stretch(var=stretch)` which would be broken by this change. So I don't think we can do this change until 3.0 since we released 2.0.0rc1 and rc2 with a fixed API surface we're stuck with this name for now.

Reply:
I was originally hoping this PR would land earlier, but I've renamed `stretch` to `var` in both signatures for now: 7bba59064d854bf3f115f0186ca5430380b27656